tests/specifications/external_spec_test.py

```diff
@@ -0,0 +1,45 @@
+from fontbakery.checkrunner import Section
+from fontbakery.fonts_spec import spec_factory
+
+spec_imports = ((
+    "fontbakery.specifications",
+    (
+        "general",
+        "cmap",
+        "head",
+        "os2",
+        "post",
+        "name",
+        "hhea",
+        "dsig",
+        "hmtx",
+        "gpos",
+        "gdef",
+        "kern",
+        "glyf",
+        "prep",
+        "fvar",
+        "shared_conditions",
+    ),
+),)
+
+
+def check_filter(checkid, font=None, **iterargs):
+  if checkid in (
+      "com.google.fonts/check/035",
+      "com.google.fonts/check/036",
+      "com.google.fonts/check/037",
+      "com.google.fonts/check/038",
+      "com.google.fonts/check/039",
```

Comment:
please add comments to each of these lines with the name of the external tool being disabled